sdks/python/apache_beam/runners/google_cloud_dataflow/internal/clients/dataflow/__init__.py

```diff
@@ -20,8 +20,8 @@
 
 import pkgutil
 
-from apitools.base.py import *
-from apache_beam.internal.clients.dataflow.dataflow_v1b3_client import *
-from apache_beam.internal.clients.dataflow.dataflow_v1b3_messages import *
+from apache_beam.runners.google_cloud_dataflow.internal.clients.dataflow.dataflow_v1b3_messages import *
```

Comment:
imports are excluded from the line length lint. This should not result in a lint warning. I can divide this up into two lines, but I think it is better to keep imports in a single line.